lib/Doctrine/DBAL/Types/DefaultTypes.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Types;
+
+/**
+ * Default built-in types provided by Doctrine.
+ */
+final class DefaultTypes
+{
+    public const ARRAY                = 'array';
+    public const BIGINT               = 'bigint';
+    public const BINARY               = 'binary';
+    public const BLOB                 = 'blob';
+    public const BOOLEAN              = 'boolean';
+    public const DATE_MUTABLE         = 'date';
+    public const DATE_IMMUTABLE       = 'date_immutable';
+    public const DATEINTERVAL         = 'dateinterval';
+    public const DATETIME_MUTABLE     = 'datetime';
+    public const DATETIME_IMMUTABLE   = 'datetime_immutable';
+    public const DATETIMETZ_MUTABLE   = 'datetimetz';
+    public const DATETIMETZ_IMMUTABLE = 'datetimetz_immutable';
+    public const DECIMAL              = 'decimal';
+    public const FLOAT                = 'float';
+    public const GUID                 = 'guid';
+    public const INTEGER              = 'integer';
+    public const JSON                 = 'json';
+    public const OBJECT               = 'object';
+    public const SIMPLE_ARRAY         = 'simple_array';
+    public const SMALLINT             = 'smallint';
+    public const STRING               = 'string';
+    public const TEXT                 = 'text';
+    public const TIME_MUTABLE         = 'time';
+    public const TIME_IMMUTABLE       = 'time_immutable';
+
+    /** @deprecated json_array type is deprecated, use {@see BuiltinTypes::JSON} instead. */
```

Comment:
Comments still refer to `BuiltinTypes`

Reply:
Thanks, PhpStorm refactoring didn't catch these.